app/models/case/sar/offender.rb

```diff
@@ -47,8 +47,11 @@ def searchable_fields_and_ranks
                  late_team_id: :integer
 
   enum subject_type: {
-    offender: 'offender',
-    ex_offender: 'ex_offender',
+    offender: I18n.t('helpers.label.offender_sar.subject_type.offender'),
+    ex_offender: I18n.t('helpers.label.offender_sar.subject_type.ex_offender'),
+    detainee: I18n.t('helpers.label.offender_sar.subject_type.detainee'),
+    ex_detainee: I18n.t('helpers.label.offender_sar.subject_type.ex_detainee'),
+    probation_service_user: I18n.t('helpers.label.offender_sar.subject_type.probation_service_user'),
   }
```

Comment:
I think I see why you've done this, but I don't think the value is used in these enums - or am I missing something? :) 

If I'm not, for consistency, could we keep it the way it's done for other enums in the app? 

e.g. `ex_offender: 'ex_offender',` and so on?

Reply:
Happy to change back to ex_offender: 'ex_offender', as the value is not used.